test/libsolidity/syntaxTests/viewPureChecker/inline_assembly_instructions_allowed_pure.sol

```diff
@@ -85,6 +85,6 @@ contract C {
 // ====
 // EVMVersion: >=london
 // ----
-// Warning 5740: (94-1755): Unreachable code.
-// Warning 5740: (1768-1780): Unreachable code.
-// Warning 5740: (1823-1832): Unreachable code.
+// Warning 5740: (94-1759): Unreachable code.
+// Warning 5740: (1772-1784): Unreachable code.
+// Warning 5740: (1827-1836): Unreachable code.
```

Comment:
There are some weird CI failures. One is about this file but it complains that it's missing these:
```
  TypeError 2527: (1247-1257): Function declared as pure, but this expression (potentially) reads from the environment or state and thus requires "view".
  TypeError 2527: (1271-1288): Function declared as pure, but this expression (potentially) reads from the environment or state and thus requires "view".
```
Not sure why since the instructions that would trigger it are now commented-out.